scripts/portconfig

```diff
@@ -124,6 +124,13 @@ class portconfig(object):
     def set_fec(self, port, fec):
         if self.verbose:
             print("Setting fec %s on port %s" % (fec, port))
+        supported_fecs = self.get_supported_fecs(port)
+        if fec not in supported_fecs:
+            if supported_fecs:
+                print('fec not in {}'.format(supported_fecs))
```

Comment:
suggest also display the input FEC in the msg, the same comment to the below.

Reply:
Will fix this one but what is `the below`?